frontend_tests/node_tests/settings_org.js

```diff
@@ -542,6 +548,18 @@ test("set_up", ({override, override_rewire}) => {
         $stub_message_content_edit_limit_parent,
     );
 
+    const $stub_move_within_stream_limit_parent = $.create("<stub move_within_stream_limit parent");
+    $("#id_realm_move_messages_within_stream_limit_seconds").set_parent(
+        $stub_move_within_stream_limit_parent,
+    );
+
+    const $stub_move_between_streams_limit_parent = $.create(
+        "<stub move_between_streams_limit parent",
+    );
+    $("#id_realm_move_messages_between_streams_limit_seconds").set_parent(
+        $stub_move_between_streams_limit_parent,
```

Comment:
There is too much repetition of this code. I think we can either mock the function which uses these elements and `find` and `parent` functions or we can extract a function and call it for different settings.